packages/suite/src/views/settings/general/Language.tsx

```diff
@@ -97,9 +97,13 @@ export const Language = () => {
         >
             <TextColumn
                 title={<Translation id="TR_LANGUAGE" />}
-                description={<Translation id="TR_LANGUAGE_DESCRIPTION" />}
+                description={
+                    LANGUAGES[language].type === 'community' && (
```

Comment:
What about extraction `LANGUAGES[language].type === 'community'` to constant outside JSX (above `return (`) and use it on both places to enhance readability?

Reply:
9b7024d2dff7da0bbcdd28fa0e9e648cee24cbe1